source/benchmarking/odm-benchmarking.md

```diff
@@ -368,14 +368,19 @@ supports.
 ### Benchmark Server
 
 The MongoDB ODM Performance Benchmark must be run against a MongoDB replica set of size 1 running the latest stable
-database version without authentication or SSL enabled.
+database version without authentication or SSL enabled. The Benchmark should be run on the established internal
```

Comment:
Can we just add a sentence saying that the "latest stable" MongoDB version should be patch-pinned and updated in standalone commits as needed in order to be able to differentiate between performance profile changes due to server vs the ODM code?